Sources/OpenAPIRuntime/Documentation.docc/Documentation.md

```diff
@@ -68,15 +68,29 @@ Please report any issues related to this library in the [swift-openapi-generator
 - ``Configuration``
 - ``DateTranscoder``
 - ``ISO8601DateTranscoder``
+- ``MultipartBoundaryGenerator``
+- ``RandomMultipartBoundaryGenerator``
+- ``ConstantMultipartBoundaryGenerator``
+- ``IterationBehavior``
+
+### Content types
+- ``HTTPBody``
+- ``Base64EncodedData``
+- ``MultipartBody``
+- ``MultipartRawPart``
+- ``MultipartPart``
+- ``MultipartDynamicallyNamedPart``
 
 ### Errors
 - ``ClientError``
 - ``ServerError``
 - ``UndocumentedPayload``
 
 ### HTTP Currency Types
-- ``HTTPBody``
```

Comment:
IMO I'd keep HTTPBody here.

Reply:
It serves both as the HTTP currency type and also as the concrete type for raw bytes and plain text bodies, so I think it also makes sense under content types.

Added to both, here it makes sense for the type to be curated under two sections, to reflect the two places people might encounter it.